lib/view/resources/definitive_view.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+
+import '../../hard_data.dart';
+import '../../strings.dart';
+import '../../style.dart';
+import '../../widget/reusable_card.dart';
+
+class DefinitiveView extends StatelessWidget {
+  static const _cardHeight = 40.0;
+  static const _cardElevation = 1.0;
+
+  final cards = <Widget>[
+    ReusableCard(
+      title: Strings.informationWhCoronaTitle,
+      height: _cardHeight,
+      elevation: _cardElevation,
+      routeTo: whCoronavirusInfoURL,
+    ),
+    ReusableCard(
+      title: Strings.informationWhAnaestheticTitle,
+      height: _cardHeight,
+      elevation: _cardElevation,
+      routeTo: whAnaestheticMicrositeURL,
+    ),
+    ReusableCard(
+      title: Strings.informationWhIcuTitle,
+      height: _cardHeight,
+      elevation: _cardElevation,
+      routeTo: whIcuMicrositeURL,
+    ),
+  ];
+
+  static Widget _buildSpacer({double height = 16}) => Container(height: height);
```

Comment:
could we name `16` as const, seems to be used in more than one place in this file